server/mail/mail.go

```diff
@@ -238,7 +238,11 @@ func (m mailService) sendMail(e fleet.Email, msg []byte) error {
 	}
 
 	if err := client.Quit(); err != nil {
-		return fmt.Errorf("error on client quit: %w", err)
+		// Ignore EOF errors on quit, which can happen if the server
+		// closes the connection after the message is sent.
+		if err.Error() != "EOF" {
```

Comment:
Nit: you should check `errors.Is(err, io.EOF)` instead of based on the error message.

Reply:
ah good call, updated